tree.yaml

```
@@ -55,3 +55,52 @@ products:
       HALO-20240907a.zarr: QmbbJzYxtXZa4BXoojKN9dYmR7sszypqJutYTTVdNQazy6
       HALO-20240909a.zarr: QmcQeqGbHfsUWsnZVBV1fbe9HKsd72kTgYoXzShE5SqNwb
 raw:
+  HALO:
+    radar:
+      moments:
+        HALO-240704a:
+          20240704_1312.nc: QmVXLt5xcf2tKrViCAxmqwAk2tBexSztmey5zciudsxATh
+        HALO-240709a:
+          20240716_0742.nc: QmRW17XS6Rk28eJwkHghLBrebRza8ovzmgTfDSiM2rnfKF
+        HALO-240711a:
+          20240711_0742.nc: QmS94qVJvQKPKvXKN4mk1wcReD3ph6VGNoD8T24RrVFhAX
+          20240711_1215.nc: QmRuSrNjxRsjvZ2Py9vGcuBiYU2rHYBhVFbiLa5V3in6pU
+        HALO-240716a:
+          20240716_0742.nc: QmRW17XS6Rk28eJwkHghLBrebRza8ovzmgTfDSiM2rnfKF
+        HALO-240717a:
+          20240717_0735.nc: QmcmzpS2v1vPR1S7cKVXHUo8666quTfNo9e6T9L2sFcsHA
+        HALO-240718a:
+          20240718_1001.nc: QmV6JS8YMJNmLG1wFC8RDypdyCQUtpBMuLaqxi1vDMH8p2
+        HALO-240729a:
+          20240729_0946.nc: QmeEkM8Fq3x2goyAyWsHChigr7dgqKWBW5meGmuAHy69xK
+        HALO-240730a:
+          20240730_0829.nc: QmTp1fz5oUsJWzrDWeHHPpxcXowj7wwUccjz2rtvaqsg7y
```

Diese Ordner sehen nicht nach einer Flight-Id aus. Sollte das so aussehen:?
```suggestion
        HALO-20240704a:
          20240704_1312.nc: QmVXLt5xcf2tKrViCAxmqwAk2tBexSztmey5zciudsxATh
        HALO-20240709a:
          20240716_0742.nc: QmRW17XS6Rk28eJwkHghLBrebRza8ovzmgTfDSiM2rnfKF
        HALO-20240711a:
          20240711_0742.nc: QmS94qVJvQKPKvXKN4mk1wcReD3ph6VGNoD8T24RrVFhAX
          20240711_1215.nc: QmRuSrNjxRsjvZ2Py9vGcuBiYU2rHYBhVFbiLa5V3in6pU
        HALO-20240716a:
          20240716_0742.nc: QmRW17XS6Rk28eJwkHghLBrebRza8ovzmgTfDSiM2rnfKF
        HALO-20240717a:
          20240717_0735.nc: QmcmzpS2v1vPR1S7cKVXHUo8666quTfNo9e6T9L2sFcsHA
        HALO-20240718a:
          20240718_1001.nc: QmV6JS8YMJNmLG1wFC8RDypdyCQUtpBMuLaqxi1vDMH8p2
        HALO-20240729a:
          20240729_0946.nc: QmeEkM8Fq3x2goyAyWsHChigr7dgqKWBW5meGmuAHy69xK
        HALO-20240730a:
          20240730_0829.nc: QmTp1fz5oUsJWzrDWeHHPpxcXowj7wwUccjz2rtvaqsg7y
```

---

Maybe these are also garbage data from ground days. We probably only want to start on 20240809 (maybe with the addition of the scientific test flight before).